tests/forte/data/data_store_test.py

```diff
@@ -647,6 +647,26 @@ def test_get_entry_attribute_by_class(self):
                 attribute_result, entry_name_attributes_dict[entry_name]
             )
 
+    def test_is_subclass(self):
+        self.assertTrue(
+            self.data_store._is_subclass(
+                "data_store_test.TitleTest", Annotation
+            )
+        )
+
+        self.assertFalse(
+            self.data_store._is_subclass(
+                "data_store_test.TitleTest", MetricTest
+            )
+        )
+        # ``Generics`` is the grandfather of "data_store_test.SingleMetricTest" class.
+        self.assertTrue(
+            self.data_store._is_subclass(
+                "data_store_test.SingleMetricTest", Generics
+            )
+        )
+
+
```

Comment:
Add test in:
1. query class that already exists in type_attributes (you can add more info into the type_attributes for testing)
2. get every ancestors of a class
3. test with the flag `no_dynamic_subclass` on

Reply:
Looks like 3 tests suggested are included. Resolving this.